dali/base/dadfs.cpp

```diff
@@ -5306,9 +5312,11 @@ class CDistributedSuperFile: public CDistributedFileBase<IDistributedSuperFile>
         init(_parent,tree,_name,user,transaction);
     }
 
-    CDistributedSuperFile(CDistributedFileDirectory *_parent, IPropertyTree *_root)
+    CDistributedSuperFile(CDistributedFileDirectory *_parent, IPropertyTree *_root, const char *optionalName)
     {
         commonInit(_parent, _root);
+        if (optionalName)
+            logicalName.set(optionalName);
```

Comment:
Arguably that should throw an error in CDfsLogicalFileName::set()
Rather than bail out here silently.
I'm sure there are other places that *could* potentially pass "" to CDfsLogicalFileName::set()

I think a separate issue / another PR for a very rainy day.